universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -355,16 +357,25 @@ private ContainerTag blockEditPanel(
   }
 
   private ContainerTag renderPredicate(
-      PredicateDefinition predicate,
+      long programId,
+      long blockId,
+      Optional<PredicateDefinition> predicate,
       String blockName,
       ImmutableList<QuestionDefinition> questions) {
+    String currentBlockStatus =
+        predicate.isEmpty()
+            ? "This screen is always shown."
+            : predicate.get().toDisplayString(blockName, questions);
     return div()
         .withClasses(Styles.M_4)
         .with(
             div("Visibility Condition").withClasses(Styles.TEXT_LG, Styles.FONT_BOLD, Styles.PY_2))
+        .with(div(currentBlockStatus).withClasses(Styles.TEXT_LG, Styles.MAX_W_PROSE))
         .with(
-            div(predicate.toDisplayString(blockName, questions))
-                .withClasses(Styles.TEXT_LG, Styles.MAX_W_PROSE));
+            redirectButton(
+                ReferenceClasses.EDIT_PREDICATE_BUTTON,
+                "Edit Visibility Condition",
```

Comment:
nit: Update this to use Sentence case, like "Edit visibility condition" - same with "Visibility condition" above. It's a small thing, but good for keeping our casing consistent across the site. :)

Reply:
Thanks so much for catching that!